charts/tidb-operator/templates/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- if .Values.admissionController.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: admission-controller-certs
+data:
+  cert.pem: {{ .Values.admissionController.cert }}
+  key.pem: {{ .Values.admissionController.key }}
```

Comment:
You can create cert's secret like [this](https://github.com/goharbor/harbor-helm/blob/master/templates/ingress/secret.yaml)

Reply:
With this method, the key，cert and ca will be updated each time you upgrade.